perf.groovy

```diff
@@ -37,3 +55,61 @@ def projectFolder = Utilities.getFolderName(project) + '/' + Utilities.getFolder
         }
     }
 }
+
+// Create the Linux/OSX/CentOS coreclr test leg for debug and release and each scenario
+[true, false].each { isPR ->
+    ['Ubuntu'].each { os ->
+        def osGroup = getOSGroup(os)
+        def architecture = 'x64'
+        def configuration = 'Release'
+        def newJob = job(Utilities.getFullJobName(project, "perf_${osGroup}", isPR)) {
+            steps {
+                // Set up the copies
+                // Coreclr build containing the tests and mscorlib
+                shell('curl http://dotnet-ci.cloudapp.net/job/dotnet_coreclr/job/master/job/release_windows_nt/lastSuccessfulBuild/artifact/bin/tests/tests.zip -o tests.zip')
```

Comment:
@jashook This is just a replacement for "copyArtifacts" that works cross server.  I think it could go either way as to whether to hoist or not

However, the project + branch in the addresses should no be hard coded.  These should be pulled from the variables available in the script.  Otherwise the netci.groovy becomes non-portable across branches and forks